dev/build/windows/makecoq_mingw.sh

```diff
@@ -928,7 +928,7 @@ function make_camlp5 {
 function make_lablgtk {
   make_ocaml
   make_findlib
-  make_camlp4 # required by lablgtk-2.18.3 and lablgtk-2.18.5
+  make_camlp5 # required by lablgtk-2.18.3 and lablgtk-2.18.5
```

Comment:
Watch out that the Windows build scripts do actually require camlp4 to build lablgtk.